diff --git a/package/Config.in.host b/package/Config.in.host
index 5bb9894df1..a6ccf75b07 100644
--- a/package/Config.in.host
+++ b/package/Config.in.host
@@ -30,6 +30,7 @@
 	source "package/dos2unix/Config.in.host"
 	source "package/dosfstools/Config.in.host"
 	source "package/doxygen/Config.in.host"
+	source "package/dropbear/Config.in.host"
 	source "package/dracut/Config.in.host"
 	source "package/dtc/Config.in.host"
 	source "package/e2fsprogs/Config.in.host"
diff --git a/package/dropbear/Config.in.host b/package/dropbear/Config.in.host
new file mode 100644
index 0000000000..57b263ae74
--- /dev/null
+++ b/package/dropbear/Config.in.host
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_HOST_DROPBEAR
+	bool "host dropbearkey"
+	help
+	  Dropbearkey can generate the SSH keys on the host if they have
+	  to be included in the Buildroot output file. This feature can
+	  be required for the targets where a persistent location to
+	  store these keys is not available, new keys being otherwise
+	  generated at each boot.
diff --git a/package/dropbear/dropbear.mk b/package/dropbear/dropbear.mk
index 36eb7898ba..61051c314f 100644
--- a/package/dropbear/dropbear.mk
+++ b/package/dropbear/dropbear.mk
@@ -135,4 +135,9 @@ define DROPBEAR_INSTALL_TARGET_CMDS
 	ln -snf /var/run/dropbear $(TARGET_DIR)/etc/dropbear
 endef

+HOST_DROPBEAR_MAKE = \
+	$(MAKE) \
+	PROGRAMS="dropbearkey"
+
 $(eval $(autotools-package))
+$(eval $(host-autotools-package))
